.gitignore

```diff
@@ -15,4 +15,4 @@ python_env
 .ropeproject/
 .pytest_cache/
 venv/
-
+.python-version
```

Comment:
Was handy for me to ignore this file created by pyenv. Has nothing to do with the rest of the changes.